awx/main/tasks/system.py

```diff
@@ -240,15 +240,10 @@ def apply_cluster_membership_policies():
         logger.debug('Cluster policy computation finished in {} seconds'.format(time.time() - started_compute))
 
 
-@task(queue='tower_broadcast_all')
+@task(queue='tower_settings_change')
 def clear_setting_cache(setting_keys):
-    orig_len = len(setting_keys)
-    for i in range(orig_len):
-        for dependent_key in settings_registry.get_dependent_settings(setting_keys[i]):
-            setting_keys.append(dependent_key)
-    cache_keys = set(setting_keys)
-    logger.debug('cache delete_many(%r)', cache_keys)
-    cache.delete_many(cache_keys)
+    # log that cache is being cleared
+    logger.info(f"clear_setting_cache of keys {setting_keys}")
```

Comment:
This needs to contain the original logic with `delete_many` because that's how the settings actually get cleared. Meaning, the method _contents_ need to be what they were before.

Reply:
Thank you. Missed that I hadn't added it back.